base/loading.jl

```diff
@@ -419,8 +420,9 @@ function require(mod::Symbol)
             cachefile = compilecache(mod)
             m = _require_from_serialized(1, mod, cachefile, last)
             if isa(m, Exception)
-                warn("Compilecache failed to create a usable precompiled cache file for module $name. Got:")
+                warn("`compilecache` failed to create a usable precompiled cache file for module $name. Got:")
```

Comment:
I agree a convention like this would be useful. `warn` could print expressions between backticks in bold/bright to make it clearer that these are identifiers. Actually, my first PR against Julia (https://github.com/JuliaLang/julia/pull/4762) was about using `'` to mark identifiers in messages, but @JeffBezanson wasn't happy with that convention.